examples/browser-api-playground/src/components/Layout/PageLayout.tsx

```diff
@@ -1,18 +1,36 @@
-import { PropsWithChildren } from "react";
+import { PropsWithChildren, useEffect, useState } from "react";
 import Header from "./Header";
-import { EmbedConfigContext } from "../../EmbedConfigContext";
+import { EmbedConfigContext, sdkOptionsKey } from "../../EmbedConfigContext";
 import useConfigStore from "../../useConfigStore";
 import { Layout } from "antd";
 import { Content } from "antd/es/layout/layout";
 
 const PageLayout = ({ children }: PropsWithChildren) => {
+  const [ready, setReady] = useState(false)
   const contextValue = useConfigStore()
 
+  useEffect(() => {
+    try {
+      if (!contextValue.config?.[sdkOptionsKey]?.source) {
+        throw new Error('No source URL provided')
+      } else {
```

Comment:
I guess this else statement is not required.

Reply:
ya can remove